atlp-rwanda/atlp-pulse-fn
# redesign all components using new theme

# PR Description
New UI implementation

# Description of tasks that were expected to be completed

- [x] Theme is applied across all existing pages/components
- [x] Landing page is implemented in light theme

# How has this been tested?

- Checkout to branch `ft-new-ui-implementation`
- Check UI implementation

# Screenshots (If appropriate)

<img width="1440" alt="Screen Shot 2023-08-03 at 12 12 04" src="https://github.com/atlp-rwanda/atlp-pulse-fn/assets/113624105/9d50f77d-69ca-4a76-8fdc-50b8cd77b68f">
<img width="1440" alt="Screen Shot 2023-08-03 at 12 12 20" src="https://github.com/atlp-rwanda/atlp-pulse-fn/assets/113624105/f5358961-5fdf-443e-b0f8-41e159afa42c">
<img width="1440" alt="Screen Shot 2023-08-03 at 12 12 32" src="https://github.com/atlp-rwanda/atlp-pulse-fn/assets/113624105/fed8cbc3-4f56-4eac-90b1-58617059ebda">

# Please check this Checklist before you submit your PR:
